### DISCUSSION TITLE
Too Frequent updates on Jetbrains IDEs and when update is ready, obsolete version stops working.

### DISCUSSION THREAD

**Discussion by tislib** [community/community#74998] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Hi Copilot team.
It is very great what you did and I am very happy to use this product. But one thing becoming annoying is the updates.
I tend to update things on time, but not as soon as it become available.
The problem with Jetbrains IDEs plugins are, whenever update is available, existing installed plugin stops working. And it is forcing me to update to continue using copilot.
It is happening in the middle of my work so either I need to stop my work update/restart IDE or stop using copilot for a while.
I think that older versions should work or updates should not be that frequent.

**Comment by Akash1134** [comment]
Hi @tislib ,

Thank you for sharing your feedback and we will pass it on to our engineers to take a closer look at.

**Comment by Ares9323** [comment]
I agree with this, but I think it's more a problem with the JetBrains suite that doesn't allow auto-update in background like vscode.
There's an [open feature request](https://youtrack.jetbrains.com/issue/IDEA-185204) if you want to upvote it.
Copilot updates are definitely too frequent, but you can't really complain if a developer is very active, usually the issue is the opposite!

**Comment by wldhg** [comment]
JetBrains' plugin loading method is definitely a problem, but I think the main issue is the update frequency of Copilot. Despite being a paid program, Copilot is updated every few hours, and its changelog seems to be neglected. I can't distinguish whether I'm using the stable version or the nightly version, and I'm even doubtful if proper Q/A is being conducted.

**Comment by tislib** [comment]
Hi everyone. 
I think that, the problem is, when new version comes, previous version become broken, and users are forced to update to use plugin. Basically, without update, it is not working.

**Comment by Murodxojayev1ch** [comment]
Response #1

**Comment by khaledh** [comment]
I have never seen a plugin updated so frequently. I mean it's good that improvements are being made, but there needs to be a balance between pushing every little update to users (requiring a restart of the IDE) and going stale for a long period of time.

**Comment by maryamSayagh** [comment]
update: 08/2025 and we still have to restart everyday mid work
